bucket/liquidsoap.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.4.3",
+    "description": "Audio and video streaming language",
+    "homepage": "https://www.liquidsoap.info",
+    "license": "GPL-2.0",
```

Comment:
Invalid SPX identifier

Reply:
Their license didn't specify whether it was GPL-2.0-only or GPL-2.0-or-later, so it's now GPL-2.0-only unless they change/ specify it.

Reply:
Nope. On second thought it makes more sense to include later versions.

> This program is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation; either version 2 of the License, or
    (at your option) any later version.